nanshe_workflow/par.py

```diff
@@ -123,6 +129,71 @@ def get_client(profile):
     return(client)
 
 
+@contextmanager
+def get_executor(client):
+    executor = client.become_dask()
+    try:
+        yield executor
+    finally:
+        executor.shutdown()
```

Comment:
Seems there is a [`stop_dask` method]( http://ipyparallel.readthedocs.io/en/latest/api/ipyparallel.html?highlight=stop_dask#ipyparallel.Client.stop_dask ) for the `ipyparallel.Client` instance that we should have been using to do cleanup instead.